### DISCUSSION TITLE
help getting 8x 1080Ti (12gb) working together

### DISCUSSION THREAD

**Discussion by kon75** [oobabooga/text-generation-webui#4257] [Q&A]
came across text-generation-webui as I have 8x 1080Ti cards I'm hoping could be fully utilized.
installed via start-windows.bat. everything worked and I was able to access the local site without any errors.
under model I installed TheBloke_falcon-7b-instruct-GPTQ, model loader is AutoGPTQ, the interface sees all 8x GPUs but I had to slide the bar on each to the right showing 11200 now. cpu-memory in MiB is 81800. wbits 4, groupsize 64, only other option checked is 'trust-remote-code'. in the chat tab I have 'instruct' selected on the button.

something is definitely wrong with my config as the performance is horrible! 15-21sec per word in the response.
what super odd is checking nvidia-smi, the GPU utilization (100%) cycles through all 8 GPUs one at a time every second.
hoping to stick with Falcon. 7B should fit NP, 40B might have to be float16 to fit?
any suggestion? please let me know if you need me to provide additional info. TY!

**Comment by kon75** [comment]
also tried using TheBloke_Llama-2-7b-Chat-GPTQ; model loader ExLlamav2_HF; GPU-split 8,10,10,10,10,10,10,10
results; according to nvidia-smi, only GPU0 is being utilized.
performance: Output generated in 84.67 seconds (1.89 tokens/s, 160 tokens, context 712, seed 479244691)
best result has been: Output generated in 15.45 seconds (4.47 tokens/s, 69 tokens, context 888, seed 1621215411)

when switching to model loader AutoGPTQ, more GPUs appears to be utilized but ranging from 0% to 39%.
performance was Output generated in 99.00 seconds (1.57 tokens/s, 155 tokens, context 72, seed 2136741057)

still seems a bit short of what 8x GPUs should be able to do, even being 1080Ti. Thoughts?

**Comment by Tedy50** [comment]
I think with so many GPUs in series a lot of time is consumed on communication and probably all your GPUs are running in 1x pcie mode which makes everything extra slow
